venia-integration-tests/src/tests/integrationTests/wishList/verifyMultipleWishlistFeatures.spec.js

```diff
@@ -60,8 +60,8 @@ const {
     getNewCustomerWishlistCall
 } = graphqlMockedCallsFixtures;
 
-// TODO add tags MOS, AC to test to filter and run tests as needed
-describe('PWA-1782: verify single wishlist basic features', () => {
+
+describe('PWA-1782: verify single wishlist basic features', { tags: ['@commerce', '@open-source', '@ci'] },() => {
```

Comment:
Multiple wishlists is a commerce only feature.

Reply:
Thanks @fooman , updated tags.